### ISSUE TITLE
xplique: handle pytorch and onnx

### ISSUE THREAD

**Issue by lucashervier** [deel-ai/xplique#29]
As a general improvment, we would like to extend the library so that you can work with the PyTorch Framework and the ONNX open standard format.

**Comment by lucashervier** [comment]
Solved in PR #59. In addition the documentation refactor in the PR #79 should make it clearer on how to make it possible. This is only available for BlackBox attribution methods since the WitheBox attributions needs a gradient access with TensorFlow
